packages/devtools/web/main.dart

```diff
@@ -20,5 +21,10 @@ void main() {
     }
   });
 
+  if (!browser.isChrome) {
+    framework.showWarning('WARNING: Unsupported browser; DevTools is only '
```

Comment:
Users are unfortunately blind to warning messages.
Make this the only content that is displayed and require the user to click somewhere to continue to use the app in an unsupported browser or don't even give users that option as things are likely badly broken.
In the spirit of error message improvement, tweak the message to something like:
ERROR: You are running devtools with $browserName but DevTools only runs on Chrome.

Reopen this url in a Chrome browser to use devtools.

<link>Continue to use devtools in this unsupported browser</link>

What I like about this general form is the first line covers the problem and indicates the solution. I'm not great at writing error messages so feel free to weak or ignore my suggestion.